misc/alert_rules.json

```diff
@@ -30,5 +30,13 @@
   {
     "rule": "%ports.ifOperStatus = \"down\" && %ports.ifOperStatus_prev = \"up\" && %macros.device_up = \"1\"",
     "name": "Port status change from up to down"
+  },
+  {
+    "rule": "%ports.ifOutErrors_rate >= \"100\" || %ports.ifInErrors_rate >= \"100\"",
```

Comment:
Ok no worries. The only other issue with this is that || don't work well right now so it needs to be two rules or done as a macro